integration_test/tests/common/mod.rs

```diff
@@ -179,13 +179,16 @@ pub fn connect_and_allocate_statement(
     in_connection_string: Option<String>,
 ) -> (HDbc, HStmt) {
     let conn_str = in_connection_string.unwrap_or_else(generate_default_connection_str);
-    let conn_handle = connect_with_conn_string(env_handle, conn_str).unwrap();
+    let conn_handle = connect_with_conn_string(env_handle, Some(conn_str)).unwrap();
```

Comment:
You don't need this logic here anymore if you want to make it an option.
This is a spot where you can use `None`.

Reply:
I think we want to pass the `in_connection_string` argument since it's already an `Option<String>`!